kolibri/plugins/coach/assets/src/state/store.js

```diff
@@ -54,15 +54,26 @@ const mutations = {
   SET_MODAL(state, modalName) {
     state.pageState.modalShown = modalName;
   },
+
   ADD_GROUP(state, group) {
     state.pageState.groups.push(group);
   },
-  UPDATE_GROUP(state, groupId, updatedGroup) {
-    state.pageState.groups.forEach((group, index, arr) => {
-      if (group.id === groupId) {
-        arr[index] = updatedGroup;
-      }
-    });
+  RENAME_GROUP(state, groupId, newGroupName) {
```

Comment:
> So you are suggesting to for example instead of this, have a UPDATE_GROUPS that is passed in the updated groups array?

That does seem a little more consistent with our typical use of mutations/actions